modules/order/templates/commerce-order-receipt.html.twig

```diff
@@ -0,0 +1,127 @@
+{#
+/**
+ * @file
+ * Template for the order receipt.
+ *
+ * Available variables:
+ * - order: The order entity.
+ * - billing_information: Billing information for the order.
+ * - shipping_information: Shipping information for the order.
```

Comment:
We have the billing_information / payment_method split. Wouldn't it make sense to have shipping_information AND shipping_method to parallel that?